src/misc/folderDetails.ts

```diff
@@ -97,6 +99,36 @@ export class FolderDetails {
         }
     }
 
+    public async matchAppsEngineVersion(): Promise<void> {
```

Comment:
Shouldn't we ask them which version they prefer? Or just make the decision for them?

Reply:
Yes, right now we are forcing this new version

Reply:
I agree. If they realize they want a different Apps-Engine version, they can run NPM install again with that version (and deal with the potential compiler errors)